tests/kafkatest/tests/benchmark_test.py

```diff
@@ -26,16 +28,22 @@
 DEFAULT_RECORD_SIZE = 100  # bytes
 
 
-class Benchmark(KafkaTest):
+class Benchmark(Test):
```

Comment:
Why did we need to switch this from inheriting from `KafkaTest` to `Test`? We'd really like to avoid redundant code. The vast majority of tests really should be implemented via `KafkaTest`. Do we just need to generalize `KafkaTest` a bit more?